ivy/functional/ivy/experimental/manipulation.py

```diff
@@ -1593,3 +1594,41 @@ def expand(
         Output Array
     """
     return ivy.current_backend(x).expand(x, shape, out=out)
+
+
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+@handle_array_function
+def concat_from_sequence(
+    input_sequence: Union[ivy.Array, ivy.NativeArray],
```

Comment:
This is a sequence of arrays; the type-hint should be similar to what you did for the backends.